client/spring-zeebe/src/main/java/io/camunda/zeebe/spring/client/annotation/JobWorker.java

```diff
@@ -9,7 +9,10 @@
 @Target(ElementType.METHOD)
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
-public @interface ZeebeWorker {
+public @interface JobWorker {
```

Comment:
Rather than renaming, I would duplicate and mark the ZeebeWorker as deprecated  with a link to the new annotation explaining the changing behaviors.

Reply:
Same remark for other annotations :)

Reply:
+100! The idea with that branch was simply to show the target.
Really doing it involves some more work also to make sure deprecated anootations are still processes normally.

Reply:
wonderfull :)